ml/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
   api("ai.h2o:h2o-ext-target-encoder:${h2oVersion}")
   api("ai.h2o:mojo2-runtime-api:${mojoPipelineVersion}")
   api("ai.h2o:mojo2-runtime-impl:${mojoPipelineVersion}")
+  api(project(':sparkling-water-api-generation'))
```

Comment:
@mn-mikke Since I'm using `IgnoredParameters` class in the code, I need to include the generation code into the assembly.

To be honest, I do not like this that much. The other approach I was thinking about was to copy the IgnoredParameters class to ml src-gen directory via gradle copy task so we do not need to have such dependency. WDYT?

Reply:
I don't see a big problem with that. The dependency was there already, but as compile-only [line 31](https://github.com/h2oai/sparkling-water/pull/2131/files#diff-0e661b343c1a53714aa5ee80b5db48a8L31). 

Btw, can you use `implementation` instead of `api`?

Reply:
oki oki, thanks.

Will change

Reply:
Done :)